27_tictactoe/lib/player.py

```diff
@@ -0,0 +1,11 @@
+
+class Player:
+
+    def __init__(self, name, marker):
+        if type(name)!=str: raise ValueError('Player name must be a string')

```

Comment:
Type checking is not necessary here: You can always try to convert to string, and if it fails python automatically raises an exception